data/OGEmanuel.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Ogunmola Emmanuel",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "English major turned **Tech bro**. Love to code, eat, watch football and watch movies",
+  "avatar": "https://github.com/OGEmanuel.png",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/its_GODwithMe",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/OGEmanuel",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/emmanuel-ogunmola-a5007a1a6/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Gmail",
+      "url": "emanuolumide@gmail.com",
```

This link won't work. `mailto` must be used for this to work.
```suggestion
      "url": "mailto:emanuolumide@gmail.com",
```